pymc_marketing/mmm/tvp.py

```diff
@@ -146,7 +257,7 @@ def create_time_varying_intercept(
         )
         return pm.Deterministic(
             name="intercept",
-            var=intercept_base * multiplier,
+            var=pt.exp(intercept_base + multiplier),
```

Comment:
I like this parametrization much better!

Reply:
Hey team, can we deep-dive a bit more about this implementation?

In the original implementation, The mean of 𝛽𝑡 would be the 
`**_beta` (since the mean of multiplier is 1). Then variance of 𝛽𝑡 will depend on the variance of the `multiplier`.

Because of it, the initial implementation is more straightforward and interpretable (Personal opinion). Mostly around the response curves.

1. What would be the benefit of this parameterization, if it is not more interpretable? 
2. Being exponential, it forces the final parameter to be positive. This depending on the parameter we want to be time dependent may or may not be needed.
3. I think that if the result of `**_base + latent` is a normal distribution, there should be no problem, the result is a LogNormal distribution. But if it is `Gamma` or `HalfNormal`, the result would be an even more skewed distribution -This might be feasible because the config controls the base distributions- I'm not sure if it's a problem or not, but have we thought about how this transformation would apply to different arbitrary distributions?

A couple of questions that may be very obvious and I am a little slow to see the answers! 😅